### PR TITLE
Add heading rules to varia based themes

### PR DESCRIPTION
**Pull request by scruffian**
This is a different approach to https://github.com/Automattic/themes/pull/2551 which fixes all Varia based themes.

### CODE REVIEW

File: varia/style-editor.css [hunk 1]
```diff
@@ -727,6 +727,14 @@ object {
 	margin-bottom: 0;
 }
 
+.wp-block-group {
+	/* Treating H2 separately to account for legacy /core styles */
+}
```
Comment (MaggieCabrera) [inline varia/style-editor.css:732]: Are we getting this empty rule because of the comment?

Comment (scruffian) [reply varia/style-editor.css]: seems like it. interestingly switching to a `//` comment removes it...

**Comment by MaggieCabrera** [comment]
I was trying to review but I'm not sure what the correct behavior is. A heading with no defined alignment goes to the right on RTL site, the rest goes to the alignment that was selected in the editor, is that correct? If so, then this looks fine to me:

Screenshot 2020-11-03 at 18 38 17

File: varia/style-editor.css (hunk 1 above)
Comment (allancole) [reply varia/style-editor.css]: Yeah, I think the Sass compiler completely removes `//` but it keeps `/* … */`. 

_I believe_ there’s a separate, later script on wpcom that minifies each theme stylesheet which removes all code comments in CSS.

**Comment by scruffian** [comment]
Looking at this some more I think it's a bad idea, see https://github.com/Automattic/themes/pull/2551#issuecomment-721419680

**Comment by allancole** [comment]
This is what I’m getting on my end with **RTL turned on** inside of a group block:

image

Looks like the alignment is still getting reversed in RTL, and _I think_ its supposed to stay persistent regardless of the RTL setting.

image

For other text-align utility class rules we’re using a Sass flag to ignore the RTL so the floating position isn’t automatically flipped, which breaks the expected behavior. See here:

https://github.com/Automattic/themes/blob/96c77a2a4b3e38e6a2ae1581459845525c6fc23e/varia/sass/blocks/utilities/_style.scss#L6-L10

**Comment by scruffian** [comment]
I wonder if we can just remove these rules and let core handle it all?

**Pull request closed without merging**